DFE-Digital/Dfe.Academies.External
# 103590 loans

Resolves:-
https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_workitems/edit/103590/
https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_workitems/edit/103858/

## Type of change


## Expected behaviour

You can progress the loan section under finances and the loans will be sent to the API

Added loan functionality to the school finances section





## Steps to test this PR

1. Go to the finance review page
2. Fill out the loans section
3. Confirm the data is saved in the database

## Prerequisites
n/a
